tests/MMS/hw/runtest

```diff
@@ -17,7 +19,9 @@ print("Making MMS test")
 shell_safe("make > make.log")
 
 # List of NX values to use
-nxlist = [16, 32, 64, 128, 256, 512]
+nxlist = [32, 64]
+# Old list:
+# nxlist = [16, 32, 64, 128, 256, 512]
```

Comment:
Again, I'm not sure about this, as it makes the test less rigorous

Reply:
To some extend yes - as we are not going to such high grid numbers, but it does not change the outcome and makes it much faster.

Reply:
I checked:

`nx_max` | convergence | wall time
-----|----|---
16 | 1.593060 | ~1.5 s
32 | 1.940418 | ~ 3 s
40 | 1.964195 | ~ 4 s
48 | 1.976635 | ~ 5 s
64 | 1.988427 | ~10 s
128 | 1.998536 | ~ 45 s
256 | 2.000325 | ~ 640 s

Not sure why we should actually run with more than `nx=40`.
128 is an order of magnitude more precise then we check for in any test ...
I guess it looks nicer when you can plot a longer line showing the conversion, but I think to actually use in a test suite, this is perfectly reasonable.

Reply:
The issue is not when it works -- it's when it doesn't work. What we normally see with MMS tests working correctly is that the error scaling is not right at very low grid resolution (as you would expect) as well as at high grid resolution (due to a different dominant error source), and it's only in some middle range that the error scaling holds. One way failures manifest themselves is by a shortening of this range from either or both ends. So what we want to check is that the error scales correctly over a wide range of reasonable grid resolutions.

The MMS tests are slow -- but that's fine, this PR lets us run them optionally. This is something we'll only do infrequently, possibly with a cron job on Travis or something, or just for releases.

Reply:
In that case we should make sure that it converges over the broad range, not just the last point.
I can have a go when I have more time ...

Right now, this might still be an improvement, as it will at least detect if the range is shortened from one end, rather than to check somewhere in the middle (I guess)